src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
```

Comment:
Can you fix the order of this class too? Again, all others are alphabetical.

These should be `static` classes, correct?